---
layout: page
title: Research
permalink: /research/
nav: true
nav_order: 2
horizontal: false
---

A major challenge in human genetics is determining which noncoding sequences act as enhancers or promoters, which genes they regulate and through which mechanisms, and in which cell types these activities are present. 

<br>
<div class="row justify-content-sm-center">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.liquid loading="eager" path="assets/img/EP_graphic.jpg" %}
    </div>
</div>
<div class="caption">
From Andersson, R., Sandelin, A. <a href="https://doi.org/10.1038/s41576-019-0173-8">Determinants of enhancer and promoter activities of regulatory elements</a>. Nat Rev Genet 21, 71–87 (2020)
</div>

Regulation of gene expression is one of life’s most fundamental biological processes and it influences all aspects of human biology. Consequently, dysfunction of gene regulation has emerged as a central mechanism in the pathogenesis of human diseases. Human genome-wide association studies (GWAS) have discovered hundreds of thousands of associations between genetic variants and human diseases or phenotypic traits. A majority of these associations involve non-coding variants that overlap gene regulatory elements, in particular enhancers and promoters. However, it has proven challenging to connect GWAS associations to their causal mechanisms, because we do not fully understand the logic by which our genome encodes for regulatory activity and cell-type specific functions, and how these activities are mediated by regulatory proteins. Therefore, there is a great need for accurate computational models of gene regulation that can be used to interpret the impact of genetic variants on molecular phenotypes and how these translate to disease risk. 

<br>
<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        {% include figure.liquid loading="eager" path="assets/img/challenges.jpg" %}
    </div>
</div>
<div class="caption">
From Zaugg, J.B., Sahlén, P., Andersson, R. et al. <a href="https://doi.org/10.1038/s41594-022-00896-3">Current challenges in understanding the role of enhancers in disease</a>. Nat Struct Mol Biol 29, 1148–1158 (2022)
</div>

---

<h3>Research aims</h3>

To characterize fundamental gene regulatory mechanisms and gain insights into molecular mechanisms by which enhancer or promoter dysregulation contributes to disease risk, we focus on developing computational tools, predictive models, and experimental assays to:

- Determine which genomic sequences act as enhancers or promoters
- Map enhancer-gene regulatory interactions
- Determine the cell types in which these activities are present
- Learn how the human genome encodes regulatory activity
- Determine the function of regulatory genetic variants

---

<h3>Representative work</h3>

- Andersson R, et al. <a href="http://dx.doi.org/10.1038/nature12787">An atlas of active enhancers across human cell types and tissues</a>. Nature. 2014
- Rennie S, et al. <a href="http://dx.doi.org/10.1093/nar/gky244">Transcription start site analysis reveals widespread divergent transcription in D. melanogaster and core promoter-encoded enhancer activities</a>. Nucleic Acids Research. 2018
- Einarsson H, et al. <a href="https://doi.org/10.7554/eLife.80943">Promoter sequence and architecture determine expression variability and confer robustness to genetic variants</a>. Elife. 2022
- Wenger, et al. <a href="https://doi.org/10.1038/s41588-023-01476-x">Symmetric inheritance of parental histones governs epigenome maintenance and embryonic stem cell identity</a>. Nature Genetics. 2023
- Sheth, Qiu, et al. <a href="https://doi.org/10.1101/2024.11.23.624931">Mapping enhancer-gene regulatory interactions from single-cell data</a>. bioRxiv. 2024
